ui.frontend/src/view/FormFieldBase.js

```diff
@@ -105,9 +105,11 @@ export default class FormFieldBase extends FormField {
         this.updateReadOnly(state.readOnly)
         this.updateEnabled(state.enabled, state)
         this.initializeHelpContent(state);
+        this.registerWidgetEventListeners();
```

Comment:
applyState is not a good place for registering listeners.
this method is meant for applying model state to the markup.